src/utils.rs

```diff
@@ -323,6 +325,24 @@ pub fn selection_ranges(root: &SyntaxNode, content: &str, pos: Position) -> Opti
     root.map(|b| *b)
 }
 
+pub fn atom_edit_to_tuple(atom_edit: AtomEdit) -> (ops::Range<usize>, String) {
+    let r: std::ops::Range<usize> = atom_edit.delete.start().into()..atom_edit.delete.end().into();
+    (r, atom_edit.insert.to_string())
```

Comment:
Would be nice to document what this is doing and also to add a test.

Reply:
Good idea, done in 4cc254aab2e565c2d9f9b83fa9d8f36eea1a2ba2.